frontend/src/common/enums/app/app-route.enum.ts

```diff
@@ -2,6 +2,7 @@ enum AppRoute {
   ROOT = '/',
   SIGN_IN = '/sign-in',
   SIGN_UP = '/sign-up',
+  CREATE_PODCAST = '/create-podcast',
```

Comment:
```suggestion
  EDIT_PODCAST = '/podcasts/edit/:id',
```
we'll use this page for creating & editing
(check documentation)